examples/tenant.yaml

```diff
@@ -181,6 +181,15 @@ spec:
   #   value: "off" # to turn-off browser
   # - name: MINIO_STORAGE_CLASS_STANDARD
   #   value: "EC:2"
+  #   ## For secure env vars like passwords, create an opaque Kubernetes secret and specify the secret in 
+  #   ## the `valueFrom` field. The `valueFrom` object must contain the following fields:
+  #   ## - `name` - the secret from which MinIO extracts the password
+  #   ## - `key` - the data field within secret, whose value will be set to the env variable's value
+  # - name: MINIO_IDENTITY_LDAP_LOOKUP_BIND_PASSWORD
```

Comment:
This doesn't match the example, unless I'm missing some nuance?

Reply:
which example?

Reply:
I think it's a stray line - we already define what `name` is on line 186, and then re-define it here as if it's a static value.